app/assets/javascripts/controllers/live_metrics/util/metrics-parse-url-factroy.js

```diff
@@ -1,13 +1,43 @@
 angular.module('miq.util').factory('metricsParseUrlFactory', function() {
+  var re = /([^&=]+)=?([^&]*)/g;
+  var decodeRE = /\+/g;  // Regex for replacing addition symbol with a space
+  var decode = function(str) {
+    return decodeURIComponent( str.replace(decodeRE, ' ') );
+  };
+
+  var parseParams = function(query) {
```

Comment:
@yaacov do we really need this custom parameters parser? If so do you think it will be reused in ManageIQ for other parts?
cc @himdel